.gitignore

```diff
@@ -1 +1,2 @@
 .env
+.venv
```

Comment:
You should not include __pycache__ within the git repository

Reply:
fixed